Update VSCode debugging guide (#2189)
ballerina-platform/ballerina-dev-website: learn/tooling-guide/visual-studio-code-extension/run-and-debug/run-and-debug.md

---
layout: ballerina-tooling-guide-left-nav-pages-swanlake
title: Run and Debug

> description: Describes the existing Ballerina run and debugging support in vscode 

Reviewer: ```suggestion
description: Describes the existing run and debugging functionalities provided by Ballerina in Visual Studio Code. 
```

Author: Updated in 7025c57

2. Open the Terminal and execute the Ballerina command, which you want to debug, out of the supported remote debugging commands below. 

    - Debugging a Ballerina package or a single file: 

    ```bash
    bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>
    ```

>    - Debugging Ballerina executable jar:  

Reviewer 2: ```suggestion
   - Debugging Ballerina executable:  
```

Author: Executable jar looks better to me, as using just "executable" might leave some doubt (executable code, an executable file, or an executable program?)

Author: @praneesha WDYT?

Reviewer: +1 say "executable JAR". Note the uppercase letters in the word "JAR".

Author: Updated in f8d2254

    ```bash 

>     bal run --debug <DEBUGGEE_PORT> <JAR_FILE_PATH>

Reviewer 2: ```suggestion
    bal run --debug <DEBUGGEE_PORT> <EXECUTABLE_PATH>
```

Author: https://github.com/ballerina-platform/ballerina-dev-website/pull/2189#discussion_r636828843

Reviewer: Shall we say `<EXECUTABLE_JAR_FILE_PATH>`?

Author: @praneesha +1

Author: Updated in f8d2254
